#! /bin/sh -e

# DP: add manpage for capiinit(8)

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- isdnutils-3.3.orig/capiinit/Makefile.am
+++ isdnutils-3.3/capiinit/Makefile.am
@@ -13,5 +13,6 @@
 LDADD = 
 
 sbin_PROGRAMS = capiinit
+man_MANS = capiinit.8
 
 capiinit_SOURCES = capiinit.c
--- isdnutils-3.3.orig/capiinit/capiinit.8
+++ isdnutils-3.3/capiinit/capiinit.8
@@ -0,0 +1,25 @@
+.\" $Id: $
+.\"
+.\" CHECKIN $Date: $
+.\"
+.TH capiinit 8
+.SH NAME
+capiinit \- start or stop CAPI-capable ISDN cards
+.SH SYNOPSIS
+capiinit start | stop
+.SH DESCRIPTION
+.B capiinit
+will replace avmcapictrl in the future.
+
+If
+called as "capiinit start" it will load all modules, add all cards
+configured in "/etc/isdn/capi.conf" and load the firmware to all active
+cards.
+
+When called as "capiinit stop", it will deinit all cards and
+remove all modules.
+
+There is a sample config in /etc/isdn/capi.conf .
+
+.SH AUTHOR
+manpage written from C source by Paul Slootman <paul@debian.org>.
